dot/network/stream_manager.go

```diff
@@ -23,20 +21,22 @@ type streamData struct {
 // This prevents keeping stale streams open and continuously trying to
 // read from it, which takes up lots of CPU over time.
 type streamManager struct {
-	ctx           context.Context
-	streamDataMap *sync.Map //map[string]*streamData
+	ctx                   context.Context
+	streamDataMap         *sync.Map //map[string]*streamData
```

Comment:
maybe a bit out of scope, but **SYNC.MAP** here

Reply:
SYNC.MAP??? Probably out of scope I would think. If we really are anti sync map maybe we should open an issue/epic to remove them from codebase?

Reply:
:smile: issue is #2281 and PR for that sync.Map is #2366 , feel free to review that one too!